game-core/src/test/java/games/strategy/triplea/ai/pro/util/MoveBatcherTest.java

```diff
@@ -113,25 +118,26 @@ public void testTransportsPickingUpUnitsOnTheWay() {
     //   Load tank and two infantry onto the transports.
```

Comment:
side-note, the 'favor using helper functions' could help reduce the length of this test and perhaps make it easier to understand, eg:

```
givenTransportsMovedIntoPosition();
givenTankAndInfantryLoadedOntoTransport();
givenTransportedUnitsAreMoved();
givenUnitsUnloaded();

assertThat(moves, hasSize(4));
```